README.rst

```diff
@@ -56,28 +61,22 @@ Usage Example
 
 .. code-block :: python
 
-    # This example shows using two TSL2491 light sensors attached to TCA9548A channels 0 and 1.
-    # Use with other I2C sensors would be similar.
-    import time
+    # This example shows using sensors attached to TCA9548A channels 1 and 5.
```

Comment:
Example looks like a scan. Need comment change here?

Reply:
Solved in https://github.com/adafruit/Adafruit_CircuitPython_TCA9548A/pull/35/commits/8d66db73557f009d38d79dfca13f7e8d0e8e88ff
Thanks